source/dub/packagemanager.d

```diff
@@ -297,41 +325,62 @@ class PackageManager {
 		return &iterator;
 	}
 
-
 	/** Returns a list of all package overrides for the given scope.
 	*/
-	const(PackageOverride)[] getOverrides(LocalPackageType scope_)
```

Comment:
The work is now done as member of Repository, but there's a Path-based function here to refer to the right member of m_repos.